core/src/main/java/org/jobrunr/jobs/context/JobDashboardLogger.java

```diff
@@ -64,6 +65,10 @@ public JobDashboardLogLines() {
         public void add(JobDashboardLogLine line) {
             logLines.add(line);
         }
+        
+        public void addAll(Collection<JobDashboardLogLine> lines) {
+            logLines.addAll(lines);
+        }
```

Comment:
Figured this approach was in-line with the existing `add`.
Alternatively, a secondary constructor taking an initial collection could be added.

Reply:
Let's go for the secondary constructor approach? I think it makes more sense in the long run.